Client/core/CMainMenu.cpp

```diff
@@ -429,6 +390,12 @@ void CMainMenu::Update()
         m_bHideGame = false;
     }
 
+    // Make sure frame rate limit gets applied
+    if (m_bIsIngame)
+        CCore::GetSingleton().EnsureFrameRateLimitApplied(); // Catch missed frames
+    else
+        CCore::GetSingleton().ApplyFrameRateLimit(88);       // Limit when not connected
+
```

Comment:
Why was this logic moved and why was the condition changed `if (m_pClientBase)`?